infra/stack/eks.tf

```diff
@@ -0,0 +1,69 @@
+module "label" {
+  source = "cloudposse/label/null"
+
+  name       = var.label_name
+  attributes = ["cluster"]
+}
+
+locals {
+  # Prior to Kubernetes 1.19, the usage of the specific kubernetes.io/cluster/* resource tags below are required
+  # for EKS and Kubernetes to discover and manage networking resources
+  # https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html#base-vpc-networking
+  tags = { "kubernetes.io/cluster/${module.label.id}" = "shared" }
+}
+
```

Comment:
Minor: I usually like to place external modules on a different file, like `config.tf` or similar. Helps with organization, upgrades, etc

Reply:
Sure, will do it going forward